### ISSUE TITLE
[BUG]: Privacy Cleanup/Clear (Reset) Network Data Usage is trying to delete *.dll files in Windows directory

### ISSUE THREAD

**Issue by selivan** [undergroundwires/privacy.sexy#265]
### Description

Privacy Cleanup/Clear (Reset) Network Data Usage is trying to delete *.dll files in Windows directory

### OS

Windows 11 22H2 build 22621.2361
privacy.sexy — v0.12.4

### Reproduction steps

- select [x] Privacy Cleanup/Clear (Reset) Network Data Usage
- press "Run"
- see errors in the cmd window

### Scripts

```
@echo off
:: https://privacy.sexy — v0.12.4 — Tue, 03 Oct 2023 15:55:39 GMT
:: Ensure admin privileges
fltmc >nul 2>&1 || (
    echo Administrator privileges are required.
    PowerShell Start -Verb RunAs '%0' 2> nul || (
        echo Right-click on the script and select "Run as administrator".
        pause & exit 1
    )
    exit 0
)


:: ----------------------------------------------------------
:: -------------Clear (Reset) Network Data Usage-------------
:: ----------------------------------------------------------
echo --- Clear (Reset) Network Data Usage
setlocal EnableDelayedExpansion 
    SET /A dps_service_running=0
    SC queryex "DPS"|Find "STATE"|Find /v "RUNNING">Nul||(
        SET /A dps_service_running=1
        net stop DPS
    )
    del /F /S /Q /A "%windir%\System32\sru*"
    IF !dps_service_running! == 1 (
        net start DPS
    )
endlocal
:: ----------------------------------------------------------


pause
exit /b 0
```

### Additional information

```
C:\WINDOWS\System32\srumapi.dll
Access is denied.
C:\WINDOWS\System32\srumsvc.dll
Access is denied.
C:\WINDOWS\System32\en-US\srumapi.dll.mui
Access is denied.
C:\WINDOWS\System32\en-US\srumsvc.dll.mui
Access is denied.
C:\WINDOWS\System32\migwiz\replacementmanifests\srumon-replacement.man
Access is denied.
```

**Comment by undergroundwires** [comment]
Thank you. This is unintended and bad. This will be fixed in next minor version. I guess the intention was to delete `%windir%\System32\sru\*` folder (according to linked article in the script). I will do some more research on this and make this script as secure and robust possible.

**Comment by undergroundwires** [comment]
I've done research on this. It seems like we do not need to delete any other file but `C:\System32\sru\SRUDB.dat` which is the SRUM database file  [1] [2] [3].

[1]: https://web.archive.org/web/20231004161122/https://github.com/libyal/esedb-kb/blob/main/documentation/System%20Resource%20Usage%20Monitor%20%28SRUM%29.asciidoc "esedb-kb/documentation/System Resource Usage Monitor (SRUM).asciidoc at main · libyal/esedb-kb  | github.com"
[2]: https://web.archive.org/web/20231004161132/https://security.opentext.com/appDetails/SRUM-Database-Parser "SRUM Database Parser | security.opentext.com"
[3]: https://web.archive.org/web/20231004161147/https://learn.microsoft.com/en-us/windows-server/security/windows-services/security-guidelines-for-disabling-system-services-in-windows-server#diagnostic-policy-service "Security guidelines for system services in Windows Server 2016 | Microsoft Learn"

I will do following changes:

- Change the script to delete only `SRUDB.dat` file, nothing more.
- Improve script to not attempt to start/stop the service if the file is missing.
- Add more documentation.
- Rename script to `Clear System Resource Usage Monitor (SRUM) data` for better clarity.
- Migrate script from batchfile to PowerShell for better maintainability and readability.
- Add user-friendly output messages to the script.

**Comment by undergroundwires** [comment]
The fix is released in [0.12.5](https://github.com/undergroundwires/privacy.sexy/releases/tag/0.12.5) 🚀
